Formula/scriptisto.rb

```diff
@@ -0,0 +1,18 @@
+class Scriptisto < Formula
+  desc "Language-agnostic \"shebang interpreter\" to write scripts in compiled languages"
+  homepage "https://github.com/igor-petruk/scriptisto"
+  url "https://github.com/igor-petruk/scriptisto/archive/refs/tags/v2.0.1.tar.gz"
+  sha256 "893a06d5349d2462682021f1e053488b07a608eee138dfcc9e68853223d48b81"
+  license "Apache-2.0"
+
+  depends_on "pkg-config" => :build
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    system "#{bin}/scriptisto", "new", "rust"
```

Comment:
Can we check some output here?

Reply:
I've added a test that tests the core functionality of the tool. 

I have not found any examples or anything in a cookbook that suggest that I should add any compiler as a dependency. There is a formula for `gcc`, but this test is totally happy with `clang` or `gcc` compiler, as soon as it can be called via `cc`. 

Should I add a `gcc` test dependency on Linux like below? Does it make sense for Mac OS X too, or there is a way to specify `clang` there?

```ruby
on_linux do
  depends_on "gcc" => :build
end
```

Reply:
linux has gcc pre-installed, which should be good for CI.

Reply:
And also your basic test passed ✅